src/Lykke.Service.BlockchainWallets.MongoRepositories/Wallets/CreatedTypeMappingExtensions.cs

```
@@ -0,0 +1,17 @@
+﻿using Lykke.Service.BlockchainWallets;
+
+namespace Lykke.Service.BlockchainWallets.MongoRepositories.Wallets
+{
+    public static class CreatedTypeMappingExtensions
+    {
+        public static Contract.CreatorType FromDomain(this WalletMongoEntity.CreatorTypeValues source)
+        {
+            return (Contract.CreatorType)source;
```

better to map via switch/case to throw exception, if unknown value is passed (here and below)